resource/updater.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>James Proxy</title>
+    <link href="./james.css" rel="stylesheet">
+</head>
+<body>
+    <div id="update">
+        <div class="spinner">
+            <div class="bounce-1"></div>
+            <div class="bounce-2"></div>
+            <div class="bounce-3"></div>
+        </div>
+        <div class="update-container">
+            <h2>New Version of James is available.</h2>
+            <h3>Would you like to download and install the update?</h3>
+            <div>
+                <button id="update-yes" class="btn waves-effect waves-light">Yes.</button>
+                <button id="update-no" class="btn waves-effect waves-light">No, remind me later.</button>
```

Comment:
May want to look at the [Material spec](https://www.google.com/design/spec/components/dialogs.html#dialogs-specs) for dialogs. Should flip the buttons around, so that the affirmative action (*yes*) is on the right, and use `btn-flat`.

Wondering if you should go with `Remind me later` and `Update now`, to even out the lengths so that the no action isn't so much bigger than yes.